app/forms/school_profile_form.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SchoolProfileForm
+  include ActiveModel::Model
+
+  attr_accessor :urn
+
+  validates :urn, presence: { message: "Enter a school URN" }
+
```

Comment:
Could we remove one line so that all validations are next to each other

Reply:
sure